model/encodable/keys.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/onflow/flow-go/crypto"
 )
 
+// ConsensusVoteSigLen is the length of a consensus vote as well as aggregated consensus votes.
+const ConsensusVoteSigLen = uint(crypto.SignatureLenBLSBLS12381)
+
+// RandomBeaconSigLen is the length of a randon beacon signature share as well as the random beacon resonstructed signature.
```

Comment:
```suggestion
// RandomBeaconSigLen is the length of a random beacon signature share as well as the random beacon reconstructed (i.e., aggregated) signature.
```

Reply:
I'm trying to keep the consistency of the terms, as used in the literature, but I agree this needs to be fixed across the repo:
 - random beacon --> BLS threshold signature --> reconstruction
 - staking signatures --> BLS multi signature --> aggregation

Reply:
I like Tarak's differentiation in terminology. Lets still update 
`randon` -> `random`